backend/src/prompts/templates/neo4j-graph-why.j2

```diff
@@ -0,0 +1,4 @@
+The graph represents a hierarchy starting from a Parent Account,
+which is at the top level of the graph and goes down to Transaction.
+All Transactions belong to a group or Classification and each of them 
```

Comment:
This is a little confusing to me. Could maybe be made clearer by explaining the relationships and a short explanation of each thing.